.github/workflows/ci-testing.yml

```
@@ -19,10 +19,10 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-22.04", "macos-14", "windows-2022"]
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12"]
```
Adding 3.13 doesn't seem worthwhile for now, as some functionality is currently broken on Windows.
Also, for Python 3.12, the tests appear to exceed 1 hour, which is unusual since they typically complete within 35 minutes for the other versions. Let's try running them again.